src/commands/basic/help.js

```diff
@@ -1,84 +1,94 @@
+class Help extends Command {
+  constructor(client) {
+    super(client, {
+      name: "help",
+      description: "Displays all available commands",
+      aliases: ["h", "commands"],
+      usage: ["{{p}}help", "{{p}}help ping"],
+      category: "basic",
+      dirname: __dirname,
+      enabled: true,
+      guildOnly: true,
+      memberPermissions: [],
+      accessLevel: "Member",
+      botPermissions: ["SEND_MESSAGES", "EMBED_LINKS"],
+      nsfw: false,
+      ownerOnly: false,
+      args: false,
+      cooldown: 3000,
+      requiredConfig: false,
+    })
+  }
+  async run(message, args) {
     const embed = new MessageEmbed()
-      .setColor("GREEN")
+      .setColor(this.client.config.embeds.color)
       .setAuthor(`${message.guild.me.displayName} Help`, message.guild.iconURL)
-      .setThumbnail(client.user.displayAvatarURL());
+      .setThumbnail(this.client.user.displayAvatarURL());
 
     if (!args[0]) {
       const categories = readdirSync("./commands/");
 
       embed.setDescription(
-        `These are the avaliable commands for ${message.guild.me.displayName}\nThe bot prefix is: **${client.config.prefix}**`
+        `These are the avaliable commands for ${message.guild.me.displayName}\nThe bot prefix is: **${this.client.config.prefix}**`
       );
       embed.setFooter(
-        `© ${message.guild.me.displayName} | Developed by DistroByte | Total Commands: ${client.commands.size}`,
-        client.user.displayAvatarURL()
+        `© ${message.guild.me.displayName} | Developed by DistroByte | Total Commands: ${this.client.commands.size}`,
+        this.client.user.displayAvatarURL()
       );
 
       categories.forEach((category) => {
-        const dir = client.commands.filter(
-          (c) => c.config.category === category
+        const dir = this.client.commands.filter(
+          (c) => c.help.category === category
         );
         const capitalise =
           category.slice(0, 1).toUpperCase() + category.slice(1);
         try {
           embed.addField(
             `${capitalise} [${dir.size}]:`,
-            dir.map((c) => `\`${c.config.name}\``).join(" ")
+            dir.map((c) => `\`${c.help.name}\``).join(" ")
           );
-        } catch (e) {}
+        } catch (e) {  }
       });
 
       return message.channel.send(embed);
     } else {
-      let command = client.commands.get(
-        client.aliases.get(args[0].toLowerCase()) || args[0].toLowerCase()
+      let command = this.client.commands.get(
+        this.client.aliases.get(args[0].toLowerCase()) || args[0].toLowerCase()
       );
       if (!command)
         return message.channel.send(
           embed
             .setTitle("Invalid Command.")
             .setDescription(
-              `Do \`${client.config.prefix}help\` for the list of the commands.`
+              `Do \`${this.client.config.prefix}help\` for the list of the commands.`
             )
         );
       command = command.config;
```

Comment:
Dont you need access to `command.help` which has the usage, description, name etc?